Gems/Atom/Asset/ImageProcessingAtom/Code/Source/ImageLoader/TIFFLoader.cpp

```diff
@@ -25,11 +25,13 @@ namespace ImageProcessingAtom
         static constexpr int TiffMaxMessageSize = 1024;
 
 #ifdef AZ_ENABLE_TRACING
+        // Note: the fatal errors are processed in LoadImageFromTIFF function.
+        // We only report the errors as warning here. 
         static void ImageProcessingTiffErrorHandler(const char* module, const char* format, va_list argList)
         {
             char buffer[TiffMaxMessageSize];
             azvsnprintf(buffer, TiffMaxMessageSize, format, argList);
-            AZ_Error(module, false, buffer);
```

Comment:
I'm concerned about this change, but it's opinion-based, so I'd like to get more input from others here.

This is downgrading _all_ Tiff errors to warnings. These can be anything from "failed to convert scanline" to "tiff feature is unsupported".

The argument _for_ downgrading these to warnings I believe is that as long as an output image of the correct size and type is produced, even if it has some amount of incorrect data, it should be considered successfully built, but with a warning to let you know some of it is incorrect.

The argument _against_ downgrading these to warnings is that the AP produces a lot of spammy warnings already, and it seems likely that a "not all of your image converted correctly" message could get easily lost amongst the other warnings. It also doesn't seem conceptually right (to me) to say that anything less than 100% of the pixels being converted is just a warning instead of an error.

NOTE: Even with this currently using AZ_Error, @VickyAtAZ has pointed out to me that this doesn't cause the asset build to fail, so LoadImageFromTIFFInternal would need to change to return a nullptr in the case of a TIFFReadScanline / TIFFReadTile error if we wanted tiffs with errors in one or more scanlines or tiles to fail.